mm/page_alloc.c

```diff
@@ -2422,10 +2424,11 @@ void drain_all_pages(struct zone *zone)
 	for_each_cpu(cpu, &cpus_with_pcps) {
 		struct work_struct *work = per_cpu_ptr(&pcpu_drain, cpu);
 		INIT_WORK(work, drain_local_pages_wq);
-		schedule_work_on(cpu, work);
+		queue_work_on(cpu, lru_add_drain_wq, work);
+	}
+	for_each_cpu(cpu, &cpus_with_pcps) {
```

Comment:
The extra braces seem redundant and make the diff look bigger than it really is.